### ISSUE TITLE
AutoDetectSharedResources Fails on Pristine Account

### ISSUE THREAD

**Issue by DarwinJS** [aws-quickstart/quickstart-amazon-eks#462]
![Screenshot 2022-11-23 at 6 35 57 AM](https://user-images.githubusercontent.com/11597190/203537111-a73aaf22-6abd-4a16-87c4-4dd7db682c15.png)

![Screenshot 2022-11-23 at 7 09 06 AM](https://user-images.githubusercontent.com/11597190/203543482-dccdb7e5-f36c-44ee-a95f-1e6b0ed1afec.png)

@troy-ameigh - I recall we had this problem another time and it's hard to find because taskcat does not use pristine accounts.

Someone in the SAM team was telling me that CloudFormation can create accounts now :)

**Comment by tlindsay42** [comment]
@DarwinJS Created a fresh disposable account today to test this and was not able to duplicate. I used the default parameter values except where necessary and deployed the new [VPC template](https://fwd.aws/6dEQ7) to us-east-2 using the primary production bucket in us-east-1. If you're still able to reproduce this, would you reopen this and provide some more troubleshooting information, please?

**Comment by mental-lab** [comment]
@tlindsay42 I would like to reopen this, as this is happening within my own AWS account _(never have ran the eks quickstart successfully yet due to this error)_  

I have posted the log output from the eks gather logs script to our slack channel.

**Comment by tlindsay42** [comment]
Forgot to update the thread. I've created multiple fresh disposable accounts and not able to duplicate in any of them. I realized today that I'd been testing this with the pypi version of taskcat and not the container like y'all; however, that shouldn't matter. We're still going to try to duplicating your build specs as much as possible. My best guess is still that it's a SCP permissions issue from GitHub's upstream AWS Org since I haven't been able to duplicate it in multiple types of sandbox accounts on my side and @DarwinJS can't duplicate it in AWS Event Engine accounts.

@troy-ameigh is deep diving on this today.
